elements/src/components/pos-list/pos-list.spec.tsx

```diff
@@ -70,6 +71,29 @@ describe('pos-list', () => {
     expect(el.querySelectorAll('pos-resource')).toHaveLength(2);
   });
 
+  it('renders if-typeof objects', async () => {
```

Comment:
The test name should be less technical. E.g. `renders the template for all things of the given type`